general/solomiia_tymoshchuk/Task1.java

```diff
@@ -0,0 +1,53 @@
+package general.solomiia_tymoshchuk;
```

Comment:
Your source code should be defined in the folder java, check how each solution should be defined in the file with the homework.